mission_helpers/functions/customAnimations/fn_waveAddAction.sqf

```diff
@@ -2,9 +2,14 @@ GRAD_ANIMATION_WAVE = false;
 
 _waveAction =
 [
-	'Wave both arms','Wave both arms','',
-	{[] spawn GRAD_mission_helpers_fnc_doWave;},
-	{!GRAD_ANIMATION_WAVE}
+	'waveBothArms','Wave both arms','',
+	{[] call GRAD_mission_helpers_fnc_doWave;},
+	{!GRAD_ANIMATION_WAVE},
+	{},
```

Comment:
wozu die restlichen params .. an der stelle gabs einen wirklichen bug (konnte in fahrzeugen ausgelöst werden), da wäre eine lösung schön gewesen 😄

Reply:
Das machen die Paramter.

Reply:
AH 🤣